### PR TITLE
Enhancements to FluxCD Configurations for Kubecost Actions and Carbon Costs

### PR DESCRIPTION
**Pull request by jash2105**
**Description:**

This pull request introduces significant enhancements to the FluxCD configurations to better support enterprise users by integrating the Kubecost Actions and Carbon Costs functionalities directly within the setup process. These updates are aimed at maximizing cost savings and optimizing resource management within Kubernetes environments.

**Key Enhancements:**

1. **Structured FluxCD Configurations:**
   - I've refined the `helmrelease-plugins.yaml` configuration, distinct from the `helmrelease-flux-primary.yaml` and `helmrelease-flux-secondary.yaml`, to streamline the deployment of Kubecost Actions and Carbon Costs. This approach clarifies setup procedures on the primary cluster, facilitating easier management and deployment of these features.

2. **Integration of Kubecost Actions:**
   - The configuration now includes a consolidated overview of key Kubecost Actions such as Cluster Turndown, Request Sizing, and Namespace Turndown. For each action, brief descriptions are provided with directives to refer to the official [Kubecost Actions Documentation](https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/savings/savings-actions#enabling-kubecost-actions) for detailed information.

3. **Carbon Costs Configuration:**
   - Added new configuration details on how to integrate Carbon Costs into the Allocation and Assets dashboards, emphasizing the use of KG CO2e as a metric. This section guides users to the Cloud Carbon Footprint's methodology page for comprehensive insights into carbon cost calculations.

By centralizing and enhancing the FluxCD configurations for these key features, this contribution seeks to significantly ease the implementation and management of cost-saving and environmental sustainability tools within Kubernetes clusters. 

Collaboration Note:

I welcome and encourage feedback on this contribution. Please feel free to suggest edits, ask questions, or recommend improvements.

### CODE REVIEW

**Comment by jash2105** [comment]
@jessegoodier ...Feel free to let me know about any edits / additions that are required to be made before you merge the PR!

File: fluxcd/federated-etl-flux/README.md [hunk 1]
```diff
@@ -85,6 +85,52 @@ https://docs.kubecost.com/install-and-configure/install/cloud-integration/multi-
 
 Once you have created the required secret, uncomment `kubecostProductConfigs.cloudIntegrationSecret` and populate the secret in `helmrelease-flux-primary.yaml` and apply the changes.
 
+## Kubecost Additional Plugins
+
+This section introduces additional plugins that extend the functionality of Kubecost, focusing on Kubecost Actions and Carbon Costs. These plugins provide more granular insights and control over your cloud spending and environmental impact.
+
+### Carbon Costs
+
+The Carbon Costs plugin adds a unique metric to both the Allocation and Assets dashboards within Kubecost. This metric, measured in kilograms of CO2 equivalent (KG CO2e), follows the Environmental Protection Agency (EPA)'s definition:
```
Comment (jessegoodier) [inline fluxcd/federated-etl-flux/README.md:94]:
```suggestion
The Carbon Costs plugin (also see: [official docs](https://docs.kubecost.com/using-kubecost/carbon-costs)) adds a unique metric to both the Allocation and Assets dashboards within Kubecost. This metric, measured in kilograms of CO2 equivalent (KG CO2e), follows the Environmental Protection Agency (EPA)'s definition:
```

File: fluxcd/federated-etl-flux/README.md [hunk 2]
```diff
@@ -85,6 +85,52 @@ https://docs.kubecost.com/install-and-configure/install/cloud-integration/multi-
...
+
+> **Carbon dioxide equivalent (CO2e)** is the measure used to compare the emissions from various greenhouse gases based on their global warming potential (GWP). The CO2e for any gas is calculated by multiplying the tons of the gas by the associated GWP. \(CO2e = Metric Tons x GWP of the gas\).
+
+Kubecost leverages carbon coefficient data sourced from the Cloud Carbon Footprint initiative, attributing estimated carbon costs to various assets such as disk, node, and network resources, based on their operational runtime. These costs are subsequently allocated across different resources to provide a comprehensive view of environmental impact. For deeper insights into the coefficients' methodology, refer to the [Cloud Carbon Footprint's methodology page](https://www.cloudcarbonfootprint.org/docs/methodology).
+
+
+### Kubecost Actions
+
+Kubecost Actions is a powerful feature that allows you to create and manage automated savings actions directly within your Kubecost environment. Actions enable you to optimize cost management through a variety of strategies such as cluster turndown, request sizing, and namespace turndown, among others.
```
Comment (jessegoodier) [inline fluxcd/federated-etl-flux/README.md:103]:
```suggestion
[Kubecost Actions](https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/savings/savings-actions) is a powerful feature that allows you to create and manage automated savings actions directly within your Kubecost environment. Actions enable you to optimize cost management through a variety of strategies such as cluster turndown, request sizing, and namespace turndown, among others.
```

File: fluxcd/federated-etl-flux/README.md [hunk 3]
```diff
@@ -85,6 +85,52 @@ https://docs.kubecost.com/install-and-configure/install/cloud-integration/multi-
...
+
+#### Action Capabilities
+- **Cluster Turndown:** Schedule the spin-down of clusters during idle times to save costs.
+- **Request Sizing:** Automatically right-size container requests to prevent over-provisioning.
+- **Namespace Turndown:** Manage and delete idle workloads to maintain efficiency in resource usage.
+
+#### Creating an Action
+In the Actions page, you can create a new action or manage existing ones. Choose from predefined actions like cluster turndown or request sizing, and configure them according to your needs. For experimental features like Guided Sizing and Cluster Sizing, enable them in the settings under 'Enable experimental features'.
+
+For a complete guide on how to enable and manage Kubecost Actions, including detailed configurations and recommendations, please visit the official documentation page: [Kubecost Actions Documentation](https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/savings/savings-actions#enabling-kubecost-actions).
```
Comment (jessegoodier) [inline fluxcd/federated-etl-flux/README.md:116]: I see the link here, but perhaps both linking here and the beginning makes sense?

**Pull request merged**